frontend/tests/controllers/ProblemUpdateTest.php

```diff
@@ -696,7 +696,7 @@ public function testAddOnlyPrivateTags() {
             'auth_token' => $login->auth_token,
             'problem_alias' => $problemData['request']['problem_alias'],
             'name' => 'test-tag',
-            'public' => false,
+            'public' => true,
```

Comment:
```suggestion
            'public' => false,
```

aquí por qué se cambió esto?